src/Compilers/CSharp/Test/Semantic/Semantics/Utf8StringsLiteralsTests.cs

```diff
@@ -3328,7 +3328,11 @@ class C
 }
 ";
             var comp = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.ReleaseDll);
-            var verifier = CompileAndVerify(comp, verify: Verification.Fails).VerifyDiagnostics();
+            var verifier = CompileAndVerify(comp, verify: Verification.Fails with { ILVerifyMessage = """
+[Test3]: Unexpected type on the stack. { Offset = 0x6, Found = address of Int32, Expected = Native Int }
```

Comment:
Asking the runtime team to confirm expectations here. Tracked by [https://github.com/dotnet/runtime/issues/96695#issuecomment-1883497270](https://github.com/dotnet/runtime/issues/96695#issuecomment-1883497270)

Reply:
Michal thinks this is a known roslyn codegen issue (missing `conv.u` after `ldsflda`).